backend/src/api/auth/auth.service.ts

```
@@ -0,0 +1,63 @@
+      const hashedPassword = await this.hashingService.hash(signUpDto.password);
+
+      const user = new User();
+      user.email = signUpDto.email;
+      user.password = hashedPassword;
+      user.firstName = signUpDto.firstName;
+      user.lastName = signUpDto.lastName;
+      user.phone = signUpDto.phone;
+      user.bio = signUpDto.bio;
+
+      const newUser = await this.usersRepository.save(user);
+      delete newUser.password;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)